angularjs/manageVersion/edit.directive.less

```diff
@@ -0,0 +1,5 @@
+.no-publish {
+  input.btn {
+    background: orange;
```

Comment:
@AltamashShaikh Can we use our regular orange `#f57c00` that we use in most places?

off topic: also created https://github.com/matomo-org/matomo/issues/18709 so that in future we will be able to use `@color-orange-brand`